backend/tests/integrated/infrastructures/load/adapters/test_repositories.py

```diff
@@ -0,0 +1,139 @@
+    assert all(
+        models.Data.objects.filter(
+            data__contains={"full_name": data.full_name}
+        ).exists()
+        for data in data
+    )
+
+
+def test_django_data_domain_repository_create_method_raises_custom_exception_on_database_exception(
+    mocker: MockFixture,
+    test_django_data_domain_repository: ports.AbstractDataDomainRepository,
+):
+    # Given
+    side_effect = DatabaseError
+    mocker.patch(
+        "infrastructures.apps.load.models.Data.objects.bulk_create",
+        side_effect=side_effect,
+    )
+
+    # When and then
+    with pytest.raises(common_exceptions.DatabaseError):
+        test_django_data_domain_repository.create([fake_transformed_data()])
+
+    assert not models.Data.objects.exists()
+
+
+def test_django_data_query_repository_list_method_queries_all_records(
+    test_django_data_query_repository: query_repositories.AbstractDataQueryRepository,
+):
+    # Given
+    data_number = 3
+    DataFactory.create_batch(size=data_number)
+
+    # When
+    results, count = test_django_data_query_repository.list(
+        filters=query_ports.DataFilters(),
+        ordering=query_ports.DataOrdering(),
+        pagination=pagination_dtos.Pagination(
+            offset=pagination_dtos.PAGINATION_DEFAULT_OFFSET,
+            records_per_page=pagination_dtos.PAGINATION_DEFAULT_LIMIT,
+        ),
+    )
+
+    # Then
+    assert count == data_number
+    assert isinstance(results, list)
+    assert all(isinstance(result, query_dtos.OutputData) for result in results)
+
+
+def test_django_data_query_repository_list_method_raises_custom_exception_on_database_exception(
+    mocker: MockFixture,
+    test_django_data_query_repository: query_repositories.AbstractDataQueryRepository,
+):
+    # Given
+    side_effect = DatabaseError
+    mocker.patch(
+        "infrastructures.apps.load.models.Data.objects.filter",
+        side_effect=side_effect,
+    )
+
+    # When and then
+    with pytest.raises(common_exceptions.DatabaseError):
+        test_django_data_query_repository.list(
+            filters=query_ports.DataFilters(),
+            ordering=query_ports.DataOrdering(),
+            pagination=pagination_dtos.Pagination(
+                offset=pagination_dtos.PAGINATION_DEFAULT_OFFSET,
+                records_per_page=pagination_dtos.PAGINATION_DEFAULT_LIMIT,
+            ),
+        )
+
+
+def test_django_data_query_repository_get_method_returns_detailed_record_when_record_exists(
+    test_django_data_query_repository: query_repositories.AbstractDataQueryRepository,
+):
+    # Given
+    data: models.Data = DataFactory.create()
+
+    # When
+    result = test_django_data_query_repository.get(
+        value_objects.DataId.from_hex(data.id.hex)
+    )
+
+    # Then
+    assert result.id == data.id
+    assert isinstance(result, queries.DetailedOutputData)
+
+
+@pytest.mark.parametrize(
+    ("side_effect", "expected_error"),
```

Comment:
You can't specify `expected_error`. When you are checking something in a test, it always must check one specific thing. In this case, we expect to raise custom  `Database Error`. Therefore it needs to be checked within # When and then and not passed as parameter.